config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -706,43 +706,6 @@ periodics:
           cpu: 1
           memory: "2Gi"
 
-- name: ci-benchmark-scheduler-master
```

Comment:
@Huang-Wei 
given the slack thread:
https://kubernetes.slack.com/archives/C09QZTRH7/p1651023713888349

this PR makes sense. I just wanted to ask if (given that you basically no longer maintain the legacy test:
https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/scheduler_perf_legacy_test.go

should we just remove this legacy perf tests in a subsequent PR too?

Reply:
+1 to removing the legacy tests as well.

Reply:
> should we just remove this legacy perf tests in a subsequent PR too?

I'm fine removing the legacy test files in a follow-up PR.

Reply:
btw: https://github.com/kubernetes/kubernetes/blob/master/test/integration/scheduler_perf/scheduler_test.go is also deprecated, I'm gonna remove it as well.